alchemist-incarnation-scafi/src/main/scala/it/unibo/alchemist/model/ScafiIncarnation.scala

```diff
@@ -84,12 +82,11 @@ sealed class ScafiIncarnation[T, P <: Position[P]] extends Incarnation[T, P]{
     CachedInterpreter[AnyRef](if(doCacheValue) v else v.tail, doCacheValue).asInstanceOf[T]
   }
 
-  override def createConcentration(): T = createConcentration("")
+  override def createConcentration(): T = null.asInstanceOf[T]
```

Comment:
ok to me, but that sounds a lot NPE-prone.

Reply:
Without that, there are issues about molecule initialisation.. Indeed, in `ProtelisIncarnation` `createConcentration()` returns null too ([link](https://github.com/AlchemistSimulator/Alchemist/blob/6baf5dcf38fcaa0d4ba4c3be7c6e77969fa9490c/alchemist-incarnation-protelis/src/main/java/it/unibo/alchemist/model/ProtelisIncarnation.java#L185))